plugin/storage/cassandra/factory.go

```diff
@@ -131,6 +137,262 @@ func (f *Factory) configureFromOptions(o *Options) {
 	}
 }
 
+const (
```

Comment:
please move the changes to a separate file, like `schema.go`, there's no reason for this to be in `factory.go`

Reply:
Done. Thanks